### PR TITLE
Bcloud 4021 group file

### PR DESCRIPTION
**Pull request by jo-codegirl**

### CODE REVIEW

File: Shared/BrainCloudGroupFile.mm [hunk 1]
```diff
@@ -0,0 +1,169 @@
+//
+//  BrainCloudGroup.mm
+//  brainCloudClientObjc
+//
+//  Created by Hill, Bradley on 2016-05-11.
```
Comment (lago2015) [inline Shared/BrainCloudGroupFile.mm:5]: This comment "Created by" isn't needed

File: Shared/BrainCloudGroupFile.hh [hunk 2]
```diff
@@ -0,0 +1,118 @@
+//
+//  BrainCloudGroup.hh
+//  brainCloudClientObjc
+//
+//  Created by Hill, Bradley on 2016-05-11.
```
Comment (francobithead) [inline Shared/BrainCloudGroupFile.hh:5]: This comment is not needed, please remove.

File: Shared/BrainCloudGroupFile.mm (hunk 1 above)
Comment (francobithead) [inline Shared/BrainCloudGroupFile.mm:5]: Please remove the comment with "Created by"

File: Shared/BrainCloudGroupFile.hh [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+- (void) updateFileInfo:(NSString *)groupId
+fileId:(NSString *)fileId
+version:(int)version
+newFilename:(NSString *)newFilename
+newAcl:(NSString *)newAcl
+completionBlock:(BCCompletionBlock)cb
+errorCompletionBlock:(BCErrorCompletionBlock)ecb
+cbObject:(BCCallbackObject)cbObject;
+
```
Comment (francobithead) [inline Shared/BrainCloudGroupFile.hh:114]: Need documentation added here similar to other classes.

File: Tests/TestFile.m [hunk 4]
```diff
@@ -22,25 +22,6 @@ - (void)setUp { [super setUp]; }
...
-    {
-        fputc('!', fp);
-    }
-    fseek(fp, 0, SEEK_END);
-    long fileLen = ftell(fp);
-    fclose(fp);
-    fp = NULL;
-    return fileLen;
-}
-
```
Comment (francobithead) [inline Tests/TestFile.m:43]: Did this need to be deleted ??

**Pull request merged**